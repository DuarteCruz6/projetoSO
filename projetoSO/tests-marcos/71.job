SHOW
WAIT 3000
WAIT 3000
WRITE [(v9,75)(j8,40)(f9,40)(v1,99)]
WRITE [(s1,25)(p1,47)(a8,81)]
BACKUP
READ [j8]
WRITE [(y5,83)]
READ [a8]
WAIT 2000
BACKUP
WRITE [(s5,98)(z7,32)(c3,32)(f0,37)]
