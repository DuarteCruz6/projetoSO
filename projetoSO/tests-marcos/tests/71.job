WRITE [(r9,78)(s2,63)(h7,55)]
WRITE [(n4,12)(h5,79)(b3,53)]
WRITE [(u2,78)(w8,21)(g3,62)(e7,7)]
WRITE [(x5,44)(b2,19)(m2,9)]
WAIT 2000
WRITE [(u8,77)(w5,100)]
WRITE [(d7,17)(r1,21)(p3,17)]
SHOW
WRITE [(f4,15)(k4,61)]
WRITE [(c6,52)(g5,2)(o3,100)(i8,66)(m6,64)]
READ [g5]
WRITE [(x2,92)(j0,29)(u1,87)]
WRITE [(u4,57)(w4,30)(a9,83)(d3,58)]
DELETE [a9]
