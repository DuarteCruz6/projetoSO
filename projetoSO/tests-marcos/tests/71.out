(b2, 19)
(b3, 53)
(d7, 17)
(e7, 7)
(g3, 62)
(h5, 79)
(h7, 55)
(m2, 9)
(n4, 12)
(p3, 17)
(r1, 21)
(r9, 78)
(s2, 63)
(u8, 77)
(u2, 78)
(w5, 100)
(w8, 21)
(x5, 44)
[(g5,2)]
